libs/src/fixed_point/ifp128.sw

```diff
@@ -568,3 +568,66 @@ impl Power for IFP128 {
         }
     }
 }
+
+#[test]
+fn test_ord() {
```

Comment:
This needs to be moved to https://github.com/FuelLabs/sway-libs/blob/master/tests/src/fixed_point/ifp128_test/src/main.sw

Same applies to other tests

Reply:
Actually i was gonna say those should be moved out of there. The structure makes no sense, its a sway script thats called by a rust sdk (and thats the only thing the sdk test does). Why are inline tests in the sdk tests section?

The only CI for sway inline tests is within the libs itself so i added the tests here. Should we make a seperate inline tests section in the tests folder?